src/modules/ROOT/pages/changelog/202209/06.json

```diff
@@ -0,0 +1,206 @@
+      ],
+      "why": [
+        "because providing a handle shouldn't be a necessity.",
+        "The resource can be created without it."
+      ],
+      "how": [
+        "by removing the requirement for the handle to be provided."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/fd55cce9b" 
+      ]
+    }
+  ],
+
+  "fixed": [
+    {
+      "what": [
+        "invalid record revision history for multi value fields."        
+      ],
+      "how": [
+        "The first value appeared duplicated, the last value was omitted, and when a value was removed the value was no longer visible in the old version."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/b203224df"
+      ]
+    },
+    {
+      "what": [
+        "uninformative internal error error message on the user profile change password interface."        
+      ],
+      "how": [
+        "by properly handling and serving the error message."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/82c70b366"
+      ]
+    },
+    {
+      "what": [
+        "issues with Block title and Description when translations are included."        
+      ],
+      "how": [
+        "by updating translations and they no longer affects other blocks with duplicate titles and description are included."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/752/commits/6856d91ea09d3a78750acb60c94a456e4a09d426"
+      ]
+    },
+    {
+      "what": [
+        "being unable to export an user that had a suspended or deleted role assigned to it."        
+      ],
+      "how": [
+        "the export logic was resolving unresolved resources on the fly and it was not excluded and this edge case caused the server to panic."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/14326713e"
+      ]
+    },
+    {
+      "what": [
+        "missing support for filtering, sorting, and paging over federation nodes."        
+      ],
+      "how": [
+        "by enabling filtering, sorting, and paging for federation nodes."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/22f49b031"
+      ]
+    },
+    {
+      "what": [
+        "hiding of calendar in calendar block when switching from page builder to view page and vice versa."
+      ],
+      "how": [
+        "by replacing throttle with async/await in the CalendarBase component."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/4124f8fee4db4e546f836ff2ab9b2222cbb5c666"
+      ]
+    },
+    {
+      "what": [
+        "searching a record list if it does not have any fields selected in the configurator."
+      ],
+      "how": [
+        "by making sure a record list with default fields uses those instead of the selected ones to search from."
```

Comment:
Can we describe the issue it caused instead of saying we fixed filtering for multi-value fields? Did it throw an error or just not show anything